agentspawner/daemon.py

```diff
@@ -0,0 +1,126 @@
+#! /bin/python3
+
+"""
+Manage ideal number of "agent" like resources in Resalloc.
+"""
+
+import subprocess
+import time
+import logging
+
+from resalloc.client import (
+    Connection as ResallocConnection,
+    Ticket,
+)
+
+RESALLOC_SERVER = "http://localhost:49100"
+
+
+class SpawnerPool:
+    """
+    Manage ideal number of "agent" like resources in Resalloc.
+    """
+    sleep = 30
+
+    def __init__(self, resalloc_connection, logger):
+        self.tags = ["A"]
+        # TODO: use a persistent storage so we can restart the process
+        self.tickets = []
```

Comment:
The tickets being processed by the daemon should be stored in the database to survive service restarts.

Reply:
Can we dump the list of tickets into a persistent "text" file for now?

Reply:
It would be fine for the beginning, but I plan to use containers for deployments, so this would have to be fixed using a database before that.

Reply:
Yes, OK -> for the beginning we can have a persistent volume or a bindmounted file.